### PR TITLE
(MAINT) Fix test errors

### PR DESCRIPTION
**Pull request by jpinsonault**
Fixed test errors relating to previous PRs

### CODE REVIEW

File: test/puppetlabs/trapperkeeper/services/status/status_proxy_service_test.clj [hunk 1]
```diff
@@ -72,42 +72,42 @@
           (let [resp (http-client/get "http://localhost:8181/status-proxy/v1/services")
                 body (json/parse-string (slurp (:body resp)))]
             (is (= 200 (:status resp)))
-            (is (= {"bar" {"service_version"        "0.1.0"
+            (is (= {"bar" {"service_version" "0.1.0"
                            "service_status_version" 1
-                           "is_running"             "false"
```
Comment (jpinsonault) [inline test/puppetlabs/trapperkeeper/services/status/status_proxy_service_test.clj:77]: oof, whitespace, will fix

**Comment by KevinCorcoran** [comment]
:+1: pulled this down, merged into master, and ran tests locally.

**Comment by jpinsonault** [comment]
Fixed the whitespace ugliness

**Comment by pcarlisle** [comment]
It looks like you've solved the 503 errors by making all your test services report as running. Do you think this should have a failing service in the test or is that not important for this implementation

**Comment by jpinsonault** [comment]
That... is a good question. I'd lean towards not needing them. I think we can trust the proxy is proxying the response codes correctly.

**Comment by MSLilah** [comment]
I'm :+1:, so I'll go ahead and merge this. I also merged into master and ran the tests locally and they worked.

**Pull request merged**